### ISSUE TITLE
Invalid yaml in helm chart

### ISSUE THREAD

**Issue by maxs-rose** [openebs/lvm-localpv#269]
**What steps did you take and what happened:**

Synth yaml from cdk8s

See: https://github.com/cdk8s-team/cdk8s-core/issues/1850

**What did you expect to happen:**

Synth to work

**Anything else you would like to add:**

This is due to invalid yaml in the helm chart and cdk8s using a spec ompliant yaml parser (https://github.com/eemeli/yaml).

Example can be seen here:
https://github.com/openebs/lvm-localpv/blob/9e0ac5b4a8bacb9dc771d8d6c33293070df71507/deploy/lvm-operator.yaml#L787-L789

This needs to be indented by at least 1 space otherwise each new line of the description should be treated as a new implicit key.

```yaml
properties:
  message:
    description: 'message is a string detailing the encountered error
     during snapshot creation if specified. NOTE: message may be
     logged, and it should not contain sensitive information.'
    type: string
```

Would be valid for example

**Environment:**
- LVM Driver version: N/A
- Kubernetes version (use `kubectl version`): N/A
- Kubernetes installer & version: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from `/etc/os-release`): N/A

**Comment by maxs-rose** [comment]
I would be happy to make a MR to fix this

**Comment by Abhinandan-Purkait** [comment]
Has been included in #277
